tests/fv3_conf/ccpp_c96_HAFS_v0_hwrf_run.IN

```diff
@@ -0,0 +1,85 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+
+if [ $IMP_PHYSICS = 8 ]; then
+  if [ $WARM_START = .F. ]; then
+    cp -r @[RTPWD]/FV3_input_data_gsd/FV3_input_data_C96_with_aerosols/* INPUT/
+  else
+    cp -r @[RTPWD]/FV3_input_data_gsd/FV3_input_data_C96_with_aerosols/grid_spec*.nc INPUT/
+    cp -r @[RTPWD]/FV3_input_data_gsd/FV3_input_data_C96_with_aerosols/C96_grid*.nc INPUT/
+    cp -r @[RTPWD]/FV3_input_data_gsd/FV3_input_data_C96_with_aerosols/oro_data*.nc INPUT/
+    cp ../fv3_ccpp_gsd_coldstart${RT_SUFFIX}/RESTART/* INPUT/
+  fi
+elif [ $IMP_PHYSICS = 15 ]; then
+  if [ $WARM_START = .F. ]; then
+    cp -r @[RTPWD]/FV3_input_data/INPUT/* INPUT/
+  else
+    cp -r @[RTPWD]/FV3_input_data/RESTART/* ./INPUT/
+  fi
+else
+  echo "ERROR, no input data configured for IMP_PHYSICS=${IMP_PHYSICS}"
+  exit 1
+fi
+
+cp    @[RTPWD]/FV3_input_data/INPUT/aerosol.dat .
+cp    @[RTPWD]/FV3_input_data/INPUT/co2historicaldata_201*.txt .
+cp    @[RTPWD]/FV3_input_data/INPUT/sfc_emissivity_idx.txt .
+cp    @[RTPWD]/FV3_input_data/INPUT/solarconstant_noaa_an.txt .
+cp    @[RTPWD]/FV3_input_data/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
+cp    @[RTPWD]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
+cp    @[RTPWD]/FV3_input_data/*grb .
+cp    @[RTPWD]/FV3_input_data/*_table .
+cp    @[RTPWD]/FV3_input_data/*configure .
+
+# Copy landuse/soil/vegetation parameter tables for HWRF Noah LSM
+cp    @[RTPWD]/FV3_input_data_hafs/GENPARM.TBL .
+cp    @[RTPWD]/FV3_input_data_hafs/SOILPARM.TBL .
+cp    @[RTPWD]/FV3_input_data_hafs/VEGPARM.TBL .
+
+# Copy diag table, depending on microphysics choice
```

Comment:
so HWRF are running with different MPs?

Reply:
It runs with Thompson MP and FA-MP. It may also run with GFDL MP, but that hasn't been tested.